cmd/cephfs/main.go

```diff
@@ -32,8 +33,15 @@ var (
 	metadataStorage = flag.String("metadatastorage", "", "metadata persistence method [node|k8s_configmap]")
 )
 
+func init() {
+	klog.InitFlags(nil)
+	if err := flag.Set("logtostderr", "true"); err != nil {
```

Comment:
@Madhu-1 did you validate `logtostderr` works ?

Reply:
@humblec yes tested, am able to see  all  logs

Reply:
Thanks!

Reply:
cc @gman0